dev/fattest.simplicity/src/componenttest/rules/repeater/EE9PackageReplacementHelper.java

```diff
@@ -35,7 +35,19 @@
 
     private static final Logger Log = Logger.getLogger(EE9PackageReplacementHelper.class.getName());
 
-    public EE9PackageReplacementHelper(String rulesPath){
+    public EE9PackageReplacementHelper(String rulesPath) {
+        Properties appProps = new Properties();
+        try {
+            appProps.load(new FileInputStream(rulesPath));
+            packageRenameRules = (Map) appProps;
+        } catch (Exception e) {
+            Log.warning("Error occured when reading in " + rulesPath);
+            e.printStackTrace();
+        }
+    }
+
+    public EE9PackageReplacementHelper() {
+            String rulesPath = System.getProperty("user.dir") + "/autoFVT-templates/" + "jakarta-renames.properties";
```

Comment:
since the code here is common with the other constructor, you can just do:
```java
public EE9PackageReplacementHelper() {
  this(System.getProperty("user.dir") + "/autoFVT-templates/" + "jakarta-renames.properties");
}

Reply:
Fixed, Thank you :)